File: src/main/java/com/fifo/ticketing/domain/book/dto/BookMailSendDto.java
package com.fifo.ticketing.domain.book.dto;

import com.fifo.ticketing.domain.book.entity.BookStatus;
import java.time.LocalDateTime;
import java.util.List;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Builder
public class BookMailSendDto {

    private String emailAddr;
    private String title;

    private Long performanceId;
    private String performanceTitle;
    private LocalDateTime performanceStartTime;
    private LocalDateTime performanceEndTime;
    private String placeName;
    private List<BookSeatViewDto> seats;
    private int totalPrice;
    private int quantity;
    private BookStatus bookStatus;

}

File: src/main/java/com/fifo/ticketing/domain/book/dto/BookMailSendDto.kt
package com.fifo.ticketing.domain.book.dto

import com.fifo.ticketing.domain.book.entity.BookStatus
import java.time.LocalDateTime

data class BookMailSendDto(
    val emailAddr: String,
    val title: String,
    val performanceId: Long,
    val performanceTitle: String,
    val performanceStartTime: LocalDateTime,
    val performanceEndTime: LocalDateTime,
    val placeName: String,
    val seats: List<BookSeatViewDto>,
    val totalPrice: Int = 0,
    val quantity: Int = 0,
    val bookStatus: BookStatus,
)
